env0/utils.go

```diff
@@ -453,3 +457,34 @@ func writeResourceDataEx(prefix string, i interface{}, d *schema.ResourceData) e
 	}
 	return writeResourceDataSlice([]interface{}{i}, prefix, d)
 }
+
+func ttlToDuration(ttl string) (time.Duration, error) {
+	if ttl == "Infinite" || ttl == "inherit" {
```

Comment:
In case of `infinite` user will omit the ttl, so it will be `nil` and not `Infinite`

Reply:
Generally not an issue, but made some refactoring.
d12f08e06636e169d9c306a715ceb5f687e37d60